src/components/Pensjonsavtaler/PensjonsavtalerMobile.tsx

```diff
@@ -91,6 +91,11 @@ export const PensjonsavtalerMobil: React.FC<IProps> = ({ pensjonsavtaler }) => {
           </div>
         )
       )}
+      {(pensjonsavtaler?.length ?? 0) > 0 && (
```

Comment:
Det er allerede en sjekk på at avtaler har en length i parent komponent mener jeg:
pensjonsavtaler?.avtaler.length > 0

Trengs kanskje ikke to ganger?